Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Time/CountdownNodeable.cpp

```diff
@@ -1,90 +0,0 @@
-/*
- * Copyright (c) Contributors to the Open 3D Engine Project.
- * For complete copyright and license terms please see the LICENSE at the root of this distribution.
- *
```

Comment:
Did unit tests still pass after deleting these files and the rest of the SC files below?

Reply:
yes: https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/PR-5532/1/pipeline/295